flow/src/crud.py

```diff
@@ -192,7 +192,8 @@ async def create_message(
                     turn_id=turn_id,
                     message_type=message_type,
                     is_user_sent=is_user_sent,
-                    message=message,
+                    message=json.loads(message),
```

Comment:
@code-with-om This method expects a dict. Can we add json.loads while calling the create_message?

Reply:
This is done